doc/api/http.md

```diff
@@ -102,8 +102,8 @@ added: v0.3.4
   Can have the following fields:
   * `keepAlive` {Boolean} Keep sockets around in a pool to be used by
     other requests in the future. Default = `false`
-  * `keepAliveMsecs` {Integer} When using HTTP KeepAlive, how often
-    to send TCP KeepAlive packets over sockets being kept alive.
+  * `keepAliveMsecs` {Integer} When using HTTP KeepAlive, the initial delay 
```

Comment:
Still confusing.... the TCP keep alive socket option is not directly related to node's HTTP keepAlive option. Keep Alive packets will be sent whenever there is no data flowing over the socket for some period of time, whether HTTP keepAlive is enabled or not. Could you clarify?